src/hotspot/share/compiler/tieredThresholdPolicy.cpp

```diff
@@ -826,6 +826,18 @@ void TieredThresholdPolicy::create_mdo(const methodHandle& mh, Thread* THREAD) {
   if (mh->method_data() == NULL) {
     Method::build_interpreter_method_data(mh, CHECK_AND_CLEAR);
   }
+  if (ProfileInterpreter) {
+    MethodData* mdo = mh->method_data();
+    if (mdo != NULL) {
+      JavaThread* jt = THREAD->as_Java_thread();
+      frame last_frame = jt->last_frame();
```

Comment:
Some interpreter code is using LastFrameAccessor instead.  Does it matter here?

Reply:
No it doesn't. It's just a utility class that's used in interpreterRuntime.cpp.